.gitignore

```diff
@@ -899,3 +899,4 @@ FodyWeavers.xsd
 # End of https://www.toptal.com/developers/gitignore/api/macos,linux,windows,python,jupyternotebooks,jetbrains,pycharm,vim,emacs,visualstudiocode,visualstudio
 
 scratch/
+setup.py
```

Comment:
why would you need a setup.py? This might mean you're using a very very old version of setuptools, so I would suggest upgrading this instead of changing the .gitignore. In fact, setup.py might not even work anymore in the near future

Reply:
true, my bad